apache/geode
# GEODE-5864 Refactor pipeline deployment.

* deploy_meta.sh now looks at the file meta.properties for defaults
* Users should create meta.properties.local file and override values
  there. A common customization is to override GEODE_FORK with your
  personal geode repository.
* GRADLE_GLOBAL_ARGS is a string of gradle arguments that will be passed
  to every invocation of gradle.
* Change "start_instance.sh" to "create_instance.sh"
* Change "stop_instance.sh" to "delete_instance.sh"
* Remove unnecessary gcloud delete invocation
* Add on_failure block to rsync_code_up job to delete the instance.
* Make on_failure call a do block to be consistent with other on_failures.
* Add system property for excluding individual test classes.
* Adjust excludeTests to handle multiple tests
* Add maven snapshot bucket for storing published artifacts.
* Make GCP_PROJECT a property instead of deriving it.
* Make deploy_meta.sh more user-friendly by displaying values and
  offering the option to abort.
* Forks should no longer be hard-coded.
* Remove UpdatePassingRef job.
* Remove unused scripts.

Co-authored-by: Sean Goller <sgoller@pivotal.io>
Co-authored-by: Jacob Barrett <jbarrett@pivotal.io>
Co-authored-by: Dick Cavender <dcavender@pivotal.io>

Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:






### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.
